Sources/SwiftFormatRules/ValidateDocumentationComments.swift

```diff
@@ -95,6 +97,22 @@ public final class ValidateDocumentationComments: SyntaxLintRule {
       diagnose(.documentReturnValue(funcName: name), on: returnClause)
     }
   }
+
+  /// Ensures the function has throws documentation if it may actually throw
+  /// an error.
+  private func validateThrows(
+    _ throwsOrRethrowsKeyword: TokenSyntax?,
+    name: String,
+    throwsDesc: String?
+  ) {
+    let needsThrowsDesc = throwsOrRethrowsKeyword != nil && throwsOrRethrowsKeyword!.tokenKind == .throwsKeyword
```

Comment:
This could be written more simply as `let needsThrowsDesc = throwsOrRethrowsKeyword?.tokenKind == .throwsKeyword`; if `throwsOrRethrowsKeyword` is nil, then that equality will also fail so there's no need to check it separately.

Reply:
Fixed.